lib/src/command/get.dart

```diff
@@ -60,6 +67,50 @@ class GetCommand extends PubCommand {
     );
   }
 
+  bool shouldRunPostGetHook() {
+    final flutterRoot = Platform.environment['FLUTTER_ROOT'];
+    if (flutterRoot == null) {
+      return false;
+    }
+    if (!fileExists(p.join(flutterRoot, 'version'))) {
```

Comment:
Reuse the logic in https://github.com/dart-lang/pub/blob/9796ba9364a3168eac87362f7f222e3ccba310e9/lib/src/sdk/flutter.dart#L23 to determine if flutter is available.